molgenis-security/pom.xml

```diff
@@ -65,7 +58,7 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>nl.captcha</groupId>
+            <groupId>cn.apiclub.tool</groupId>
```

Comment:
I guess those changes (this one + other pom and dependencies changes) are part of the PR by accident?